docs/source/api_reference/transformers.rst

```diff
@@ -8,101 +8,110 @@ distribution. Real data of course is wild and hard to control. What do
 you do if you have a crazy dataset and need to bring its statistics to
 heel? Fear not for you have :code:`Transformer` objects.
 
-Transformer
```

Comment:
We should keep the references for the abstract `Transformer` superclass around. It's a useful guide for developers

Reply:
I fixed.